src/emulator/functionsEmulatorRuntime.ts

```diff
@@ -567,6 +576,7 @@ async function initializeFirebaseAdminStubs(frb: FunctionsRuntimeBundle): Promis
   // Stub the admin module in the require cache
   require.cache[adminResolution.resolution] = {
     exports: proxiedAdminModule,
+    path: path.dirname(adminResolution.resolution),
```

Comment:
This needed to be done - otherwise, the dynamic import called failed with errors like this:

```
TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received undefined
    at new NodeError (node:internal/errors:363:5)
    at validateString (node:internal/validators:119:11)
    at Object.resolve (node:path:1098:7)
    at Function.Module._nodeModulePaths (node:internal/modules/cjs/loader:625:17)
    at cjsPreparseModuleExports (node:internal/modules/esm/translators:258:30)
    at Loader.commonjsStrategy (node:internal/modules/esm/translators:188:35)
```

@samtstern Think this is safe modification, but I'm 100% unfamiliar with this code path. Let me know if this is bad (I tested things out, and emulator continues to work normally).

Reply:
This seems safe to me.